playwright/e2e/12-failed-login.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { expect } from '@playwright/test'
+
+import { test } from '../fixtures'
+
+test.beforeEach(async ({ layoutPage }) => {
+  await layoutPage.gotoHome('dashboard')
+})
+
+test.describe('Login Error Messages ', () => {
+  test('TC-01:"Should an invalid username and a valid password be entered"', async ({
+    page,
+  }) => {
+    await page.getByTestId('button-admin-login').click()
+    await page.fill('[data-testid="input-identifier"]', 'aadmin')
```

Comment:
Could follow the same approach

```suggestion
    await page.getByTestId('input-identifier').fill('aadmin')
```

But it can remain like this, we'll probably refactor it by using the locators from POM later